.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
             - uses: actions/checkout@v3
             - uses: ./.github/actions/prepare-playground
             - run: ./node_modules/.bin/cypress install --force
-            - run: npx nx affected --target=e2e --configuration=ci
+            - run: npx nx affected --target=e2e --configuration=ci --verbose
```

Comment:
This is intentional, I noticed some flaky failures on this branch that I'm yet to reproduce. Let's keep the output verbose to ease debugging of any subsequent PRs.